content/_releases/22/20.md

```diff
@@ -0,0 +1,165 @@
+
+### Sensitive action re-authentication
+
+To configure the [re-authentication for sensitive actions](#re-authentication-for-sensitive-actions):
+*   Add the `SENSITIVE_ACTION_AUTH_LIFETIME` environment variable to HELM3 secrets chart and set it to much your requirements to control how often your users must re-authenticate for sensitive actions. The system expect the value of this parameter in **milliseconds**. The default is set to `21600000` (6 hours).
+*   Enable the `tenant.featureFlags.sensitiveActionsReauth` feature flag (set it `true`).
+
+## OEM Related
+
+Information in this section is intended for our customers who use
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+
+### White-labeled credential management
+
+Integration projects for enterprise or retail solutions can become complex since there is a
+need to connect multiple endpoints within the company ecosystem. This complexity,
+oftentimes, dictates a certain rules of engagement like restricted access to the
+systems. On the other hand, unrestricted access to these system is essential for
+integration solution.
+
+From one side we have the need for integration and other side, the need for security.
+Often times, the team developing the integration is not the same team holding the
+access to these systems. To solve the dilemma we introduce **White-labeled credential management**
+feature.
+
+#### How does the White-labeled credential management work?
+
+As integrator you provide a unique URL to the credential holder to enter their access credentials without
+registering or logging-in to the system. The access holder is presented an interface
+to enter their credentials, verify them and save.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2220/RN2220-create-credential.png" title="OEM Create Credential" %}
+
+They can also modify the existing credentials as well.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2220/RN2220-modify-credentials.png" title="OEM Modify Credentials" %}
+
+For both cases they are presented with a simple screen to work with only the
+specified credentials.
+
+#### How to generate unique URL?
+
+There are certain preconditions before you can make this work:
+
+*   The end customer is already in the DB. This means you have to [create a user]({{site.data.tenant.apiDocsUri}}/v2/#create-a-user) for your credential holder, then [add him/her to the contract]({{site.data.tenant.apiDocsUri}}/v2/#add-a-new-member-to-the-contract's-scope) and [workspace]({{site.data.tenant.apiDocsUri}}/v2/#add-a-new-member-to-workspace) where the integration is set. All these actions you can do with API calls using administrative access.
+*   New created user must have only access to the credentials entity.
+*   Your tenant has SSO provider (OIDC/SAML) with `autoCreateUsers` set to `false` configuration.
+
+If the above preconditions are in place then here is how you can generate the URL
```

Comment:
Once the above conditions are met you can take the following steps to generate the URL to request credentials from their owner: